src/Microsoft.OData.Edm/ExtensionMethods/ToTraceStringExtensionMethods.cs

```diff
@@ -136,6 +137,7 @@ private static void AppendFacets(this StringBuilder sb, IEdmPrimitiveTypeReferen
             }
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = "<Pending>")]

```

Comment:
should this have something here?

Reply:
No, I should remove this line and change the codes to use the string Interpolation.  #Resolved